src/spawnAsync.ts

```diff
@@ -62,8 +62,8 @@ function spawnAsync(
       };
       if (code !== 0) {
         let error = signal
-          ? new Error(`${command} exited with signal: ${signal}`)
-          : new Error(`${command} exited with non-zero code: ${code}`);
+          ? new Error(`${command} ${args.join(" ")} exited with signal: ${signal}`)
```

Comment:
If there are no args, this will print an extra space. Needs to be fixed and a regression test added.